### PR TITLE
extended example

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Review by volodymyrss — DISMISSED** [review]

**Comment by volodymyrss** [comment]
we should report all the relevant parameter possibilities!

**Comment by burnout87** [comment]
I added a description of the possible parameters to be passed to upload a product. Please let me know if something else should be specified

**Review by volodymyrss — APPROVED** [review]
thanks! though I think example could be further extended.

**Comment by ferrigno** [comment]
You could show how to work with spectra and light curves (you have my notebooks for reference).

**Comment by volodymyrss** [comment]
You remove the output? Then the tutorial is not complete. Compare this

https://oda-api--128.org.readthedocs.build/en/128/user_guide/UploadToGallery.html

and this

https://oda-api--128.org.readthedocs.build/en/128/user_guide/GravitationalWaves.html

**Comment by burnout87** [comment]
> You remove the output? Then the tutorial is not complete. Compare this
> 
> https://oda-api--128.org.readthedocs.build/en/128/user_guide/UploadToGallery.html
> 
> and this
> 
> https://oda-api--128.org.readthedocs.build/en/128/user_guide/GravitationalWaves.html

It's not complete yet. I just updated it and coincidentally removed the output, but they will be added once the final version will be ready. 

Though the more functionality will be implemented, the more updated the notebook will need.

**Comment by volodymyrss** [comment]
> > You remove the output? Then the tutorial is not complete. Compare this
> > https://oda-api--128.org.readthedocs.build/en/128/user_guide/UploadToGallery.html
> > and this
> > https://oda-api--128.org.readthedocs.build/en/128/user_guide/GravitationalWaves.html
> 
> It's not complete yet. I just updated it and coincidentally removed the output, but they will be added once the final version will be ready.
> 
> Though the more functionality will be implemented, the more updated the notebook will need.

Ok, I see. Normally if the review is requested, and it's not in draft - I consider it ready for review. So I wait until you re-request the review when ready.

**Comment by burnout87** [comment]
Outputs are now displayed.

What else do you think we could include? Also @ferrigno if you have some more ideas.

Otherwise we can merge this for now and extend it every time a new gallery-related feature needs to be documented

**Review by volodymyrss — DISMISSED** [review]

**Comment by burnout87** [comment]
Change of the dispatcher URL to the production one still is needed

File: doc/source/user_guide/UploadToGallery.ipynb [hunk 1]
```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Quick Start (oda api v1.1.26)"
+    "# Quick Start (oda api v1.1.28)"
```
Comment (volodymyrss) [inline doc/source/user_guide/UploadToGallery.ipynb:7]: also please add it there https://github.com/oda-hub/oda_api/blob/d2426c71c141c0cc0d37b4adcf538986babcfe4c/setup.cfg#L21 so it updates automatically

**Review by volodymyrss — APPROVED** [review]

**Review by volodymyrss — APPROVED** [review]

File: tests/test_live.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def test_instruments():
         host=get_platform_dispatcher(),
         instrument="mock",
     )
-    assert disp.get_instruments_list() == ['isgri', 'jemx', 'polar', 'antares', 'spi_acs']
+    assert disp.get_instruments_list() == ['isgri', 'jemx', 'polar', 'gw', 'spi_acs', 'legacysurvey']
```
Comment (volodymyrss) [inline tests/test_live.py]: would have been better to always sort them

Comment (burnout87) [reply tests/test_live.py]: Yes, will do with the next PR.

**Pull request merged**